scripts/properties.sh

```diff
@@ -35,25 +35,35 @@ TERMUX_BASE_DIR="/data/data/${TERMUX_APP_PACKAGE}/files"
 TERMUX_CACHE_DIR="/data/data/${TERMUX_APP_PACKAGE}/cache"
 TERMUX_ANDROID_HOME="${TERMUX_BASE_DIR}/home"
 TERMUX_APPS_DIR="${TERMUX_BASE_DIR}/apps"
-TERMUX_PREFIX="${TERMUX_BASE_DIR}/usr"
+TERMUX_PREFIX_CLASSICAL="${TERMUX_BASE_DIR}/usr"
+TERMUX_PREFIX="${TERMUX_PREFIX_CLASSICAL}"
```

Comment:
I don't like this change. Can't we just append -glibc to TERMUX_PREFIX for glibc build?

Reply:
> Can't we just append -glibc to TERMUX_PREFIX for glibc build?

Well, something similar happens here - https://github.com/termux/termux-packages/pull/16901/files#diff-feabe82b48e4ea7cb1e38dddf04635abd70d3d1943332adfd218788ec36ed02aR38

This change allows: to have access to the usual `TERMUX PREFIX` ([example](https://github.com/termux-pacman/glibc-packages/blob/gpkg/gpkg/glibc-runner/build.sh)) and to simplify the logic in `build-packages.sh` scripts. Roughly speaking, this change creates two types of `TERMUX_PREFIX`: static, which in theory should not be changed for the correct operation of the compiler, and changing, which changes depending on which library it will be based on for the correct operation of the package.